content/ADR-133-scene-runtime.md

```diff
@@ -54,38 +54,41 @@ stateDiagram
+
+The exposed RPC modules are defined in the [protocol repository](https://github.com/decentraland/protocol/blob/main/proto/decentraland/kernel/apis/engine_api.proto).
+
+> TODO: define and document naming conventions about code generation for modules
 
 ```ts
-// loadModule instantiates a remote module
-declare function loadModule(moduleName: string): Promise<ModuleDescriptor>
-
-// the callRpc function takes a rpcHandle from loadModule result and a
-// methodName from the same result, combined with a list of arguments
-// it returns a promise executed in the remote module.
-declare function callRpc(rpcHandle: string, methodName: string, args: any[]): Promise<ProxyModule>
-
-declare type MethodDescriptor = { name: string }
-declare type ModuleDescriptor = {
-  rpcHandle: string
-  methods: MethodDescriptor[]
+// `require` instantiates a proxy to a RPC module. Every exposed function
+// of the module returns a promise.
+// require must fail immediately if the moduleName is invalid or unknown,
+// and it must return a Module or Proxy synchronously
+function require(moduleName: string): Module
+
+// Commonjs-compatible modules
+const exports: Object
+const module: {
+  readonly exports: typeof exports
 }
 
-// used to hook a callback to the main loop
-declare function onUpdate(callback: (deltaTimeSeconds: number) => Promise<void>): void
-```
-
-### Loading RPC modules in the scenes
+// extra functions
+function fetch(requestInit: Request): Promise<Response>
+function fetch(url: string, requestInit: Request): Promise<Response>
 
-The scene runtime exposes two asynchronous functions `loadModule` and `callRpc`. Those are the key entrypoint to communicate the scenes with other modules like the renderer. The exposed modules are defined in the [protocol repository](https://github.com/decentraland/protocol/blob/main/proto/decentraland/kernel/apis/engine_api.proto).
+class WebSocket {}
+```
 
-> TODO: define and document naming conventions about code generation for modules
+> TODO: Document fetch and WebSocket adaptations for Decentraland Scenes
 
 ### Synchronizing scene's entities with the renderer
 
-The scenes synchronize with the renderer via the `EngineAPI.SendToRenderer` RPC using the CRDT protocol defined in [ADR-117](/rpc/ADR-117). The renderer will keep a local copy of all the entities and components required for rendering. Those components are in their majority serialized using protobuf as defined in [ADR-123](/rpc/ADR-123).
+The scenes synchronize with the renderer via the `EngineApi.crdtSendToRenderer` RPC using the CRDT protocol defined in [ADR-117](/rpc/ADR-117). The renderer will keep a local copy of all the entities and components required for rendering. Those components are in their majority serialized using protobuf as defined in [ADR-123](/rpc/ADR-123).
```

Comment:
Lin to `/rpc/ADR-117` is broken and it links to ADR homepage.

Reply:
Same for ADR-123.

Reply:
Quemadaso, rpc tiró